entry_tasks/48_type_checker/type_checker.py

```diff
@@ -0,0 +1,34 @@
+if __name__ == '__main__':
+    import re
+
+    def is_not_empty(s):
+        return bool(s and s.strip())
+
+    def reformator(listik):
+        list = []
+        for item in listik:
+            if item.isdigit():
+                list.append((int(item), type(int(item))))
+            elif item.strip().replace('.', '').isdigit():
+                list.append((float(item), type(float(item))))
```

Comment:
same as above

Reply:
it is good desition: cause I found float in string '2.0' and if we wont to get output (2.0, <class 'float'>) I convert string result to float